.github/workflows/sync_to_airtable.yml

```diff
@@ -0,0 +1,59 @@
+name: sync_to_airtable
+
+on:
+  # Trigger the workflow when issues are updated
+  issues:
+    types: [opened, edited, closed, reopened, assigned, unassigned, labeled, unlabeled, milestoned, demilestoned]
+
+# Automatically cancel any prior runs of this automation
```

Comment:
Suggestion: Maybe add to this comment that this is to prevent duplicate runs (or something explaining the workflow + issue no + ref group)? In its current form, it _reads_ to me like it will cancel any previous syncs.